subsys/bluetooth/mesh/sensor.c

```diff
@@ -65,65 +51,9 @@ sensor_cadence(const struct bt_mesh_sensor_threshold *threshold,
 	}
 
 	bool in_range = BT_MESH_SENSOR_VALUE_IN_RANGE(curr, low, high);
-#endif
 	return in_range ? threshold->range.cadence : !threshold->range.cadence;
```

Comment:
This line can probably be combined with the previous, no need for a variable when the `#ifdef` is gone